xml/System.Net/WebHeaderCollection.xml

```diff
@@ -290,6 +290,11 @@
  [!code-csharp[WebHeaderCollection_Add#1](~/samples/snippets/csharp/VS_Snippets_Remoting/WebHeaderCollection_Add/CS/webheadercollection_add.cs#1)]
  [!code-vb[WebHeaderCollection_Add#1](~/samples/snippets/visualbasic/VS_Snippets_Remoting/WebHeaderCollection_Add/VB/webheadercollection_add.vb#1)]  
   
+> [!NOTE]
+> The length of `value`, that is, the string after the colon (:), is validated only in .NET Framework and .NET Core versions 2.0 - 3.1.
```

Comment:
In this overload there is no `value` parameter, it is being generated from `header`.